Userland/Utilities/unzip.cpp

```diff
@@ -76,7 +80,7 @@ static bool unpack_zip_member(Archive::ZipMember zip_member, bool quiet)
 
     if (checksum.digest() != zip_member.crc32) {
         warnln("Failed decompressing file {}: CRC32 mismatch", zip_member.name);
-        MUST(new_file->remove(zip_member.name, Core::File::RecursionMode::Disallowed, true));
+        MUST(new_file->remove(null_terminated_name, Core::File::RecursionMode::Disallowed, true));
```

Comment:
We can also drop the need for the null-terminated string here once #16636 is merged.